packages/contentful/src/crn/queries/analytics.queries.ts

```diff
@@ -269,3 +269,55 @@ export const FETCH_TEAM_COLLABORATION = gql`
     }
   }
 `;
+
+export const FETCH_ENGAGEMENT = gql`
+  query FetchEngagement($limit: Int, $skip: Int) {
+    teamsCollection(order: displayName_ASC, limit: $limit, skip: $skip) {
+      total
+      items {
+        sys {
+          id
+        }
+        displayName
+        inactiveSince
+        linkedFrom {
+          teamMembershipCollection(limit: 1) {
+            total
```

Comment:
Can a user have multiple roles in a team? From the cms that can happen but I don't know if that's a bug. We can add the same team & role multiple times to a user as well (I'm sure that's a bug though). In the first scenario, if it's not a bug wouldn't we have a wrong count?

Reply:
Nice observation, I'll ask on Slack, but I think we should not allow multiple roles 🤔